website/docs.md

```diff
@@ -1705,8 +1702,15 @@ the runtime:
    listener. In fact, the proxy balances traffic across every replica of the
    listener. (Recall that components may be replicated, and so every component
    replica will have a different instance of the listener.)
-   The proxy address is by default `localhost:0`, or the local address assigned
-   to the listener in the config file, if any.
+
+The proxy address is by default `localhost:0`, unless a concrete address
```

Comment:
Oops, I think I wrote this wrong originally. The proxy default to `":0"`, not `"localhost:0"`, right?

Reply:
Thanks, changed!